frontend/packages/topology/integration-tests/support/step-definitions/common/topology.ts

```diff
@@ -114,6 +119,19 @@ Given(
   },
 );
 
+Given(
+  'user has created workload {string} with resource type {string} and application groupings {string}',
+  (componentName: string, resourceType: string = 'Deployment', applicationGroupings: string) => {
+    createGitWorkload(
+      'https://github.com/sclorg/nodejs-ex.git',
+      componentName,
+      resourceType,
+      applicationGroupings,
+    );
+    topologyHelper.verifyWorkloadInTopologyPage(componentName);
```

Comment:
I think, we can use topologyPage.verifyWorkloadInTopologyPage(componentName);
going forward we need to remove this topologyHelper as well

Reply:
Updated